src/pages/dashboard/monitor/components/ActiveChart/index.tsx

```diff
@@ -0,0 +1,107 @@
+import React, { Component } from 'react';
+
+import { Statistic } from 'antd';
+import { MiniArea } from '../Charts';
+import styles from './index.less';
+
+function fixedZero(val: number) {
+  return val * 1 < 10 ? `0${val}` : val;
+}
+
+function getActiveData() {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [expected call-signature: 'getActiveData' to have a typedef](https://app.codacy.com/app/afc163/ant-design-pro/pullRequest?prid=4583897)